app/keepers/keepers.go

```diff
@@ -538,6 +549,14 @@ func (appKeepers *AppKeepers) SetupHooks() {
 		),
 	)
 
+	appKeepers.SwapRouterKeeper.SetPoolCreationListeners(
+		swaproutertypes.NewPoolCreationListeners(
+			// insert gamm hooks receivers here
```

Comment:
They are added right below. The comment should read as "insert swaprotuer hook receivers here". I'm just going to remove the comment to avoid confusion